lib/src/Uncertainty/Algorithm/Analytical/openturns/SORMResult.hxx

```diff
@@ -52,7 +52,10 @@ public:
   /** EventProbabilityBreitung accessor */
   Scalar getEventProbabilityBreitung() const;
 
-  /** EventProbabilityHohenBichler accessor */
+  /** EventProbabilityHohenbichler accessor */
+  Scalar getEventProbabilityHohenbichler() const;
+
+  /** EventProbabilityHohenbichler deprecated accessor */
```

Comment:
you can add a "@deprecated" doxygen comment here, plus an entry in the TODO file so I dont forget to remove in 1.16 or 1.17

Reply:
Ok. What's a doxygen comment? Could you please provide me a MWE? Just in case. Thanks.

Reply:
It's just a regular C++ comment:
```
/** @deprecated EventProbabilityHohenbichler accessor */
  Scalar getEventProbabilityHohenBichler() const;
```

Reply:
Only in the .hxx file or also in the .cxx?

Reply:
only the hxx, in the cxx theres already the logwarn

Reply:
Too late. I did it in both. Any issue with that?